lib/osf-components/addon/components/osf-navbar/x-links/hyper-link/template.hbs

```diff
@@ -7,8 +7,8 @@
     ) as |ctx|}}
         <ctx.link
             data-analytics-name='{{@analyticsLabel}}'
-            data-test-hyper-link=@testSelectorVal
-            class=@class
+            data-test-hyper-link={{@testSelectorVal}}
```

Comment:
Does that mean this never worked before?

Reply:
🤷🏾‍♂️I don't think it worked, but also there seem to be no test looking for that test selector.